Formula/mpv.rb

```diff
@@ -84,10 +82,9 @@ def install
     system "python3", "waf", "configure", *args
     system "python3", "waf", "install"
 
-    if build.with? "bundle"
-      system "python3", "TOOLS/osxbundle.py", "build/mpv"
-      prefix.install "build/mpv.app"
-    end
+    # Build the app bundle
+    system "python3", "TOOLS/osxbundle.py", "build/mpv"
+    prefix.install "build/mpv.app"
```

Comment:
Current policy is not to build .app GUI bits by default if they can be optional

Reply:
Here it's used by 40% of users, triggering build from source that can be easily avoided (and at very low cost to other users)

Reply:
Nonetheless, the eventual fate of GUI options is likely removal not default inclusion.

Reply:
Just to be sure there is no misunderstanding: mpv builds the GUI anyway. It merely wraps it in a `.app` when using the option.

Reply:
https://github.com/Homebrew/brew/blob/master/docs/Acceptable-Formulae.md#stuff-that-builds-an-app

Reply:
_“that can be distributed by Homebrew Cask”_: it won't be, because it builds from source and Cask does not like that

Reply:
mpv is already in cask.